docs/middleware/cache.md

```
@@ -10,6 +10,31 @@
+- `204: No Content`
+- `206: Partial Content`
+- `300: Multiple Choices`
+- `301: Moved Permanently`
+- `404: Not Found`
+- `405: Method Not Allowed`
+- `410: Gone`
+- `414: URI Too Long`
+- `501: Not Implemented`
+
+Additionally, `418: I'm a teapot` is not originally cacheable but is cached by this middleware.
+If the status code is other than these, you will always get an `unreachable` cache status.  
```

Markdownlint (0.37.0)

Expected: 0 or 2; Actual: 2
Trailing spaces

(MD009, no-trailing-spaces)